proto/cosmos/group/v1beta1/types.proto

```diff
@@ -162,98 +159,99 @@ message Proposal {
   // When a decision policy is changed, existing proposals from previous policy versions will become invalid.
   uint64 group_policy_version = 7;
 
-  // Status defines proposal statuses.
-  enum Status {
-    option (gogoproto.goproto_enum_prefix) = false;
+  // status represents the high level position in the life cycle of the proposal. Initial value is Submitted.
+  ProposalStatus status = 8;
 
-    // An empty value is invalid and not allowed.
-    STATUS_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "ProposalStatusInvalid"];
+  // result is the final result based on the votes and election rule. Initial value is unfinalized.
+  // The result is persisted so that clients can always rely on this state and not have to replicate the logic.
+  ProposalResult result = 9;
 
-    // Initial status of a proposal when persisted.
-    STATUS_SUBMITTED = 1 [(gogoproto.enumvalue_customname) = "ProposalStatusSubmitted"];
+  // final_tally_result contains the sums of all weighted votes for this
+  // proposal for each vote option, after tallying.
+  TallyResult final_tally_result = 10 [(gogoproto.nullable) = false];
```

Comment:
Important change 4: Rename Tally to TallyResult to match gov

Reply:
My very minor problem is that `final_tally_result` isn't necessarily final right? If you query a proposal halfway through the election you're going to get a tally_result that has some votes but not all

Reply:
After yesterday's discussion, we decided to keep `final_tally_result`, and add a tally endpoint to groups #11151